llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -3204,7 +3204,7 @@ void LoopVectorizationCostModel::collectLoopScalars(ElementCount VF) {
 
     // Determine if all users of the induction variable are scalar after
     // vectorization.
-    auto ScalarInd = llvm::all_of(Ind->users(), [&](User *U) -> bool {
+    bool ScalarInd = llvm::all_of(Ind->users(), [&](User *U) -> bool {
```

Comment:
while you are editing the line, might as well drop redundant `llvm::`
```suggestion
    bool ScalarInd = llvm::all_of(Ind->users(), [&](User *U) -> bool {
```